### DISCUSSION TITLE
Guess the song 🎸

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1056] [Galaxy Trivia] [answered]
```
Gtr. 1
|--------------------------------|-------------------------------|
|--------------------------------|-------------------------------|
|-5---------------5-----3-----5--|----------------5-----3-----5--|
|-5---------------5-----3-----5--|----------------5-----3-----5--|
|-3---------------3-----1-----3--|----------------3-----1-----3--|
|--------------------------------|-------------------------------|

|Gtr. 2
|--------------------------------|-------------------------------|
|--------------------------------|-------------------------------|
|--------------------------------|-------------------------------|
|--------------------------------|-------------------------------| 
|-3-3-3-3-3-3-3-3-3-3-3-3-3-3-3-3|3-3-3-3-3-3-3-3-3-3-3-3-3-3-...|
|--------------------------------|-------------------------------|
```
**Bonus:** Name the songwriter.

**Comment by peterbaccaro** [comment]
The song is **Eye of the Tiger** which is a well-known song by the American rock band **Survivor**. It was written by Jim Peterik and Frankie Sullivan and was released in 1982 as a single. The song gained immense popularity after being featured as the theme song for the movie "Rocky III," which starred Sylvester Stallone.

**Comment by thuongtruong1009** [comment, marked as answer]
Based on the provided guitar tablature, the song is "Smoke on the Water" by Deep Purple. The songwriter for "Smoke on the Water" is the band's guitarist, Ritchie Blackmore. The song is one of the most iconic rock riffs and a classic in the world of rock music.
